instance/models/instance.py

```diff
@@ -204,7 +204,7 @@ class GitHubInstanceMixin(VersionControlInstanceMixin):
     """
     github_organization_name = models.CharField(max_length=200, db_index=True)
     github_repository_name = models.CharField(max_length=200, db_index=True)
-    github_pr_number = models.IntegerField(blank=True, null=True)
+    github_pr_url = models.CharField(max_length=200, blank=True, default='')
```

Comment:
@antoviaque I don't think the default='' is superflous. blank=True means that the admin/form generation won't complain if the field isn't filled, but unless null=True is also set, it will still fail validation. But you don't want string fields to be null because that causes ambiguous situations when looking up unset values (is it '' or None?)

I wasn't aware of URLField. I could do that.